### PR TITLE
Improve wording: use `uncertainty` instead of `error`

### LINKED ISSUE #38: Change `withoutError` to `withoutUncert`
To be consistent, use `withoutUncert` instead of `withoutError` in the `results.tex`.

### PR DESCRIPTION
**Pull request by paul019**
Fixes #38

### CODE REVIEW

**Comment by paul019** [comment]
Fixes #38

File: src/api/console_stringifier.py
Comment (Splines) [inline src/api/console_stringifier.py:1]: Did you intend to have this in this PR?

Comment (paul019) [reply src/api/console_stringifier.py:1]: No

**Comment by Splines** [comment]
I converted this PR to a draft. Please open it once the pipeline is passing and once this PR is ready for my review.

**Review by Splines — CHANGES_REQUESTED** [review]
Could you also rename `error_name_prefix` and `error_name_suffix`?
(`Ctrl + Shift + F` in VSCode)

**Review by Splines — APPROVED** [review]

**Pull request merged**